Optimize (CQA) - Mng content - Recovering (#4874)
theforeman/foreman-documentation: guides/common/modules/proc_republishing-repository-metadata-by-using-cli.adoc

= Republishing repository metadata by using CLI

[role="_abstract"]

> Republish repository metadata when published packages do not match repository contents.

Reviewer: ```suggestion
Republish repository metadata when published packages do not match repository metadata.
```

Or, which I slightly prefer:

```suggestion
Republish repository metadata when published metadata does not match repository contents.
```

[WARNING]
====

> {Team} recommends a complete repository sync or publishing a new content view version to repair broken metadata in favor of republishing repositoriy metadata.

Reviewer: ```suggestion
{Team} recommends a complete repository sync or publishing a new content view version to repair broken metadata in favor of republishing repository metadata.
```

Typo.

= Republishing repository metadata by using CLI

[role="_abstract"]

> Republish repository metadata when published packages do not match repository contents, preferring a complete sync or new content view version when possible.

Reviewer: I suggest to shorten this, esp. in combination with the admonition below:

```suggestion
Republish repository metadata when published packages do not match repository contents.
```

[WARNING]
====
Use with caution.

> {Team} recommends a complete repository sync or publishing a new content view version to repair broken metadata instead.

Reviewer: ```suggestion
{Team} recommends a complete repository sync or publishing a new content view version to repair broken metadata in favor of republishing repositoriy metadata.
```

IMO "use with caution" is a duplicate of the admonition type. Same for Web UI.
